data/schema/types/basic.cfg

```diff
@@ -44,7 +44,7 @@
 		min=3
 		max=4
 		[element]
-			value="(?:2[0-5][0-5]|[01]?\d?\d)\s*"
+            value="(?:25[0-5]|2[0-4][0-9]|[01]?\d?\d)\s*"
```

Comment:
You changed the tabs to spaces. Please leave the indentation as is.

Reply:
hmm, i just cut and paste.  looks like my editor is changing NEW tabs to spaces